examples/static/js/ui.common.js

```diff
@@ -105,16 +105,26 @@ const formatToLogo = {
 };
 
 const trademarks = (allFormats) => {
-  let trademarknote;
-  for (const format of allFormats) {
+  let trademarknote = '';
+
+  for (let format of allFormats) {
+    let trademark = '';
+
     if (format.toLowerCase() === 'tflite') {
```

Comment:
@ibelem How about use switch statements for these three 'if (format.toLowerCase() === **format**)' blocks?

Reply:
Thanks @BruceDai , updated, PTAL :)

Reply:
LGTM, thanks